### ISSUE TITLE
v2.0.346 Prerelease completely breaks Go To Definition

### ISSUE THREAD

**Issue by JustinGrote** [dotnet/vscode-csharp#6098]
Type: <b>Bug</b>

Go To Definition is broken with latest prerelease. I highly suggest you guys publish these as github releases and turn on the "Enable Discussions for this release" so these can be reported in a more contextual manner.

https://github.com/dotnet/vscode-csharp/assets/15258962/ef1f3d15-4388-4c42-a8db-1079dfa53e00

Extension version: 2.0.346
VS Code version: Code 1.81.0 (6445d93c81ebe42c4cbd7a60712e0b17d9463e97, 2023-08-02T12:37:13.485Z)
OS version: Windows_NT x64 10.0.23451
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 6800U with Radeon Graphics          (16 x 2695)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.19GB (1.45GB free)|
|Process Argv|--folder-uri file:///c%3A/Users/jgrote/Projects/vscode-adapter --crash-reporter-id a45c1268-117e-410a-a083-cc3d7e68707f|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30803845
282f8724:30602487
89544117:30613380
showlangstatbar:30737416
vsctsb:30748421
03d35959:30757346
pythonfmttext:30731395
pythoncmv:30756943
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
gsofa:30804715
pythonnosmt12:30797651
pythonidxpt:30805730
pythonnoceb:30805159
dsvsc013:30795093
dsvsc014:30804076

```

</details>

**Comment by DoctorKrolic** [comment]
Yes, this version is completely unusable due to this bug. Please fix it ASAP

**Comment by dibarbet** [comment]
This is known and fixed in https://github.com/dotnet/vscode-csharp/pull/6091.  Hopefully will be released today.

**Comment by JustinGrote** [comment]
Confirmed fixed in 2.0.356 prerelease, thanks!
